assembl/alembic/versions/c98a9b6f6b7f_add_sections.py

```diff
@@ -49,9 +49,47 @@ def upgrade(pyramid_env):
             sa.schema.UniqueConstraint('title_id')
         )
 
-        # TODO: insert default data
+        # insert default sections
+        with m.Section.default_db.no_autoflush as db:
+            discussions = db.query(m.Discussion.id).all()
+            for discussion_id in discussions:
+                homepage_section = m.Section(
+                    discussion_id=discussion_id,
+                    title=m.LangString.create(u'Home', 'en'),
```

Comment:
for each of these, you should probably also add at least their french translations, no?

Reply:
yes, I did it in another commit